mne/preprocessing/interpolate.py

```diff
@@ -111,35 +115,64 @@ def interpolate_bridged_electrodes(inst, bridged_idx):
     bads_orig = inst.info['bads']
     inst.info['bads'] = list()
 
+    # look for group of bad channels
+    nodes = sorted(set(chain(*bridged_idx)))
+    G_dense = np.zeros((len(nodes), len(nodes)))
+    # fill the edges with a weight of 1
+    for bridge in bridged_idx:
+        idx0 = nodes.index(bridge[0])
+        idx1 = nodes.index(bridge[1])
+        G_dense[idx0, idx1] = 1
+        G_dense[idx1, idx0] = 1
+    # look for connected components
+    _, labels = connected_components(csr_matrix(G_dense), directed=False)
+    groups_idx = [
+        [nodes[j] for j in np.where(labels == k)[0]] for k in set(labels)
+    ]
+    groups_names = [
+        [inst.info.ch_names[k] for k in group_idx] for group_idx in groups_idx
+    ]
```

Comment:
This snippet looks for group of electrodes. It's equivalent to the `networkx` code:

```
G = nx.Graph()
for bridge in bridged_idx:
    G.add_edge(*bridge)
groups_idx = nx.connected_components(G)
```

I had no idea `scipy` had those functionalities, and I never used them. Maybe there is a nicer way to do this?
Also, I used `connected_components(csr_matrix(G_dense), directed=False)` as per the example of `connected_components`, but it also works without `csr_matrix`. Not sure if it's needed..